.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+## Change description
+
+> Description here
+
+## Type of change
+
+- [ ] Bug fix (fixes an issue)
+- [ ] New feature (adds functionality)
+
+## Related issues
+
+Does this PR fix a Github Issue?
+
+> Fix [#1]()
+
+## Checklists
+
+### Development
+
+- [ ] Application changes have been tested thoroughly
+
+### Security
+
+- [ ] Code follows company security practices and guidelines
+- [ ] Security impact of change has been considered
+  > Potential security impact (if any):
+
+### Code review
+
```

Comment:
I'd like to see a checklist item for adding tags so the PR will be included in our release notes (bug, enhancement, etc)